saleor/graphql/notifications/mutations/external_notification_trigger.py

```diff
@@ -0,0 +1,82 @@
+import graphene
+
+from ....core.exceptions import PermissionDenied
+from ....core.notification.mutation_handler import (
+    get_external_notification_payload,
+    send_notification,
+)
+from ....core.notification.validation import (
+    validate_and_get_external_event_type,
+    validate_and_get_payload_params,
+    validate_ids_and_get_model_type_and_pks,
+)
+from ....graphql.core.types.common import ExternalNotificationError
+from ...core.mutations import BaseMutation
+
+
+class ExternalNotificationTriggerInput(graphene.InputObjectType):
+    ids = graphene.List(
+        graphene.ID,
+        required=True,
+        description=(
+            "The list of customers or orders node IDs that will be serialized and "
+            "included in the notification payload."
+        ),
+    )
+    extra_payload = graphene.JSONString(
+        description=(
+            "Additional payload that will be merged with "
+            "the one based on the bussines object ID."
+        )
+    )
+    external_event_type = graphene.String(
+        required=True,
+        description=(
+            "External event type. In case of invalid type, "
+            "Saleor will consider the content"
+            "of that field like a ID of dynamic template."
+        ),
+    )
+
+
+class ExternalNotificationTrigger(BaseMutation):
+    class Arguments:
+        input = ExternalNotificationTriggerInput(
+            required=True, description="Input for External Notification Trigger. "
+        )
+        plugin_id = graphene.String(description="The ID of notification plugin.")
+
+    class Meta:
+        description = (
+            "Trigger sending a notification with the notify plugin method."
+            "Serializes nodes provided as ids parameter and includes this data in"
```

Comment:
```suggestion
            "Trigger sending a notification with the notify plugin method. "
            "Serializes nodes provided as ids parameter and includes this data in "
```
When constructing long strings like this you need to remember about spaces at the end of the lines. Otherwise, the entire message will be joined without them and it will be unreadable.